cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -154,6 +158,7 @@
     private final StorageClassOperator storageClassOperator;
     private final NodeOperator nodeOperator;
     private final CrdOperator<KubernetesClient, Kafka, KafkaList, DoneableKafka> crdOperator;
+    private final ZookeeperScalerProvider zooScalerProvider;
```

Comment:
Elsewhere we use `zk` as the abbreviation for ZooKeeper. I think we should be consistent.